homeassistant/components/hydrawise/binary_sensor.py

```diff
@@ -98,3 +102,10 @@ class HydrawiseBinarySensor(HydrawiseEntity, BinarySensorEntity):
     def _update_attrs(self) -> None:
         """Update state attributes."""
         self._attr_is_on = self.entity_description.value_fn(self)
+
+    @property
+    def available(self) -> bool:
+        """Set the entity availability."""
+        if self.entity_description.availability_fn:
+            return self.entity_description.availability_fn(self)
```

Comment:
```suggestion
        if self.entity_description.always_available:
            return True
```

We can always refactor the code in the future if we need to have a custom function